packages/ui-kit/src/blocks/elements/TimePickerElement.ts

```diff
@@ -0,0 +1,8 @@
+import type { Actionable } from '../Actionable';
+import type { TextObject } from '../TextObject';
+
+export type TimePickerElement = Actionable<{
+  type: 'time_picker';
+  placeholder?: TextObject;
+  initialTime?: string;
```

Comment:
What is the format of this string? Shouldn't it be more portable if it was a timestamp (`number`) instead?